### DISCUSSION TITLE
problems with sound

### DISCUSSION THREAD

**Discussion by iskandar9ll** [Whisky-App/Whisky#582] [Q&A]
cant connect headphones, even wired ones. also cant control in game sound with mac buttons.

**Comment by mavethee** [comment]
iirc wine can have issues on that point when you connect stuff on go, you would need to switch output in Wine settings manually or plug headphones before running Whisky.

`[Bottle name]` -> `Bottle Config` -> `Wine Config` -> `Sound`

**Reply by milkingmydad** [reply]
where do i find bottle config

**Reply by iskandar9ll** [reply]
![image](https://github.com/user-attachments/assets/9b12f303-526c-4bb0-bd68-cf9c94fa015f)

**Reply by milkingmydad** [reply]
ok thank you 🙏

**Reply by mavethee** [reply]
Keep in mind that WIne cannot dynamically switch between sources so either that or just connect your stuff before running Whisky.
